### PR TITLE
Rename ListViewset.list to get_list

### PR DESCRIPTION
**Pull request by pysalt**
We want to use annotation with `builtins.list` and must change list method name.

### CODE REVIEW

File: tests/app.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ class Settings(BaseSettings):
 db = Database(
     database=settings.postgres_db,
     user=settings.postgres_user,
+    password=settings.postgres_password,
```
Comment (MissiaL) [inline tests/app.py:53]: Looks like a separate commit 😃

File: freddie/viewsets/sql.py [hunk 2]
```diff
@@ -245,7 +245,7 @@ async def retrieve(self, pk: Any, *, request: Request, **params: Any) -> Model:
 
 
 class ModelListViewset(GenericModelViewSet, ListViewset):
-    async def list(
+    async def get_list(
```
[the remark this answers is not in the document]
Comment (pysalt) [reply freddie/viewsets/sql.py:248]: @arsgab what do you think about it?

[the remark this answers is not in the document]
Comment (pysalt) [reply freddie/viewsets/sql.py:248]: But in this PR I plan to change only one method that broke compatibility with python 3.9 typing. Maybe we will do it in the next releases

File: tests/app.py (hunk 1 above)
Comment (pysalt) [reply tests/app.py:53]: done

File: freddie/viewsets/sql.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (arsgab) [reply freddie/viewsets/sql.py:248]: @tsouvarev what should be consistent names for other methods? `retrieve_object`, `delete_object`?

[the remark this answers is not in the document]
Comment (pysalt) [reply freddie/viewsets/sql.py:248]: @arsgab Lets merge this PR and move this discussion to new issue

**Review by arsgab — APPROVED** [review]

**Pull request merged**